### ISSUE TITLE
Concave ethereal barrier corners without texture

### ISSUE THREAD

**Issue by InitHello** [ReikaKalseki/Reika_Mods_Issues#2036]
Not breaking anything, but I feel like a texture would improve the aesthetic.

![Example](https://i.imgur.com/GHIfln5.png)

**Comment by InitHello** [comment]
Any point in having a 'cosmetic' label for things like this?

**Comment by ReikaKalseki** [comment]
No, because that carries no actionable meaning.

**Comment by InitHello** [comment]
Indeed, I believe `not worth fixing` encompasses all of what I had in mind for a proposed `cosmetic` label.
